app/src/main/java/com/kickstarter/libs/SegmentTrackingClient.kt

```diff
@@ -142,6 +142,10 @@ open class SegmentTrackingClient(
 
             this.isInitialized = true
 
+            loggedInUser?.let {
```

Comment:
On `SegmentTrackingClient` line 58 on the initialization block you have this RX chain :

Reply:
I tested this without the `ObjectUtils.isNotNull` and the `requireNotNull` however the app still calls identify before the SDK is initialized. So on initial launch we should call identify but is ignored since the SDK is not initialized.